config/locales/et.yml

```diff
@@ -217,7 +217,6 @@ et:
         command_paper_number:
         hoc_paper_number: 'Parlamendi alamkoja dokumendi nr. '
         isbn: 'Manuse ISBN '
-        order_url: Manuse tellimise link
```

Comment:
Is this something that's used in the frontend or in whitehall-admin? It looks like user facing content, so perhaps it should be kept in?

Reply:
It is user facing but the idea is that it's not being used any more. There's now just a link to generic information that doesn't have a heading. Also from what I can tell the internationalization wasn't being used for order_url or price_in_pence so removing it doesn't really matter.